README.md

```diff
@@ -2,7 +2,7 @@
 DNS for KubeVirt VirtualMachines secondary interfaces
 
 ## Prerequisites
-1. The KubeSecondaryDNS Deployment should be reachable from outside the cluster.  
+1. The KubeSecondaryDNS Deployment should be reachable from outside the cluster via port 5353.  
```

Comment:
5353 is the internal port, not the external right ?

Reply:
It is the port that KubeSecondaryDNS is listening on to dns requests.

Reply:
it is the internal port, and the text says "from outside the cluster" so i find it misleading
because the "outside" port (or external port) is the NodePort which isn't 5353

Reply:
done